chain/consensus/filcns/filecoin.go

```diff
@@ -433,6 +436,19 @@ func (filec *FilecoinEC) VerifyWinningPoStProof(ctx context.Context, nv network.
 	return nil
 }
 
+func IsValidForSending(act *types.Actor) bool {
```

Comment:
I went carefully through this method and I don't think this needs a network version gate, because:
1. Placeholder actors cannot be created before nv18.
2. EthAccount actors cannot be created before nv18.
3. Delegated addresses cannot be created before nv18.

Note: if we sprinkle network version gates in random places, we absolutely cannot start Hyperspace with nv1. Also note that this pattern won't interact well with diverging network lineages across networks, without also including the network name in the gate.

Reply:
I will still add it, but it's entirely superfluous.